packages/apm/src/integrations/tracing.ts

```diff
@@ -122,7 +108,7 @@ export class Tracing implements Integration {
    */
   private static _getCurrentHub?: () => Hub;
 
-  private static _activeTransaction?: Span;
+  private static _activeTransaction?: SpanClass;
```

Comment:
Why do we rename in the import?

```js
import { Span as SpanClass } from '../span';
```
This file is the only one in the code base that mentions "SpanClass".

This file has yet another inconsistency which is

```js
interface Activity {
  name: string;
  span?: Span;
}
```

Why isn't the compiler or linter catching this?